common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3979,6 +3979,7 @@ public static enum ConfVars {
     HIVE_SERVER2_WEBUI_BIND_HOST("hive.server2.webui.host", "0.0.0.0", "The host address the HiveServer2 WebUI will listen on"),
     HIVE_SERVER2_WEBUI_PORT("hive.server2.webui.port", 10002, "The port the HiveServer2 WebUI will listen on. This can be"
         + "set to 0 or a negative integer to disable the web UI"),
+    HIVE_SERVER2_HA_HEALTHCHECK_PORT("hive.server2.ha.healthcheck.port", 11002, "The port the HiveServer2 health-ha web app will listen on"),
```

Comment:
Could we move this to the rest of HA configs, i.e `HIVE_SERVER2_ACTIVE_PASSIVE_HA_REGISTRY_NAMESPACE`

Reply:
Done, and also renamed the new config to HIVE_SERVER2_ACTIVE_PASSIVE_HA_HEALTHCHECK_PORT

Reply:
i don't think we need `ACTIVE_PASSIVE`, it's too long and redundant